Actions/AL-Go-Helper.ps1

```diff
@@ -686,6 +687,7 @@ function ReadSettings {
         }
         "useGitSubmodules"                              = "false"
         "gitSubmodulesTokenSecretName"                  = "gitSubmodulesToken"
+        "shortLivedArtifactsRetentionDays"              = 1  # 0 means use GitHub default
```

Comment:
I would think the default should be 0. Isn't that effectively what it is today? 

IMO the default setting for this should match whatever it is today so we don't break any partner scenarios.

Reply:
The shortlived (thisbuild) are 1 today.
The workflow using shortlived artifacts - today they don't produce artifacts - they only produce 1 day lasting thisbuild artifacts, which I wouldn't assume that anybody are fetching themselves.